Build of Paella Player 7 module fails

Building the new Paella player module fails if a local `npm` is being used, since the commands defined in the `pom.xml` do not exist:

```
❯ mvn clean install -Pdev,frontend-no-prebuilt
…
[INFO] --- exec-maven-plugin:1.6.0:exec (check) @ opencast-engage-paella-player-7 ---                                                                                                                                                         
Unknown command: "run check"                                                                                                                                                                                                                  
                                                                                                                                                                                                                                              
To see a list of supported npm commands, run:                                                                                                                                                                                                 
  npm help                                                                                                                                                                                                                                    
[ERROR] Command execution failed.                                                                                                                                                                                                             
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)                                                                                                                                                     
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)                                                                                                                                                     
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)                                                                                                                                                             
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:804)                                                                                                                                                                 
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:751)                                                                                                                                                                 
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:313)                                                                                                                                                                            
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)                                                                                                                                     
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)                                                                                                                                                       
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)                                                                                                                                                       
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)                                                                                                                                                       
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)                                                                                                                              
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)                                                                                                                               
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)                                                                                                                 
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)                                                                                                                                               
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)                                                                                                                                                                        
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)                                                                                                                                                                        
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)                                                                                                                                                                          
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)                                                                                                                                                                              
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)                                                                                                                                                                               
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)                                                                                                                                                                                 
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)                                                                                                                                                                  
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)                                                                                                                                                
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)                                                                                                                                        
    at java.lang.reflect.Method.invoke (Method.java:566)                                                                                                                                                                                      
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)                                                                                                                                                   
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)                                                                                                                                                           
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)                                                                                                                                                 
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)                                                                                                                                                             
[INFO] ------------------------------------------------------------------------                                                                                                                                                               
[INFO] Reactor Summary for opencast 12-SNAPSHOT: 
…
[INFO] Opencast :: engage-paella-player-7 ................. FAILURE [ 16.370 s]
…
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:38 min
[INFO] Finished at: 2022-04-04T21:29:52+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (check) on project opencast-engage-paella-player-7: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :opencast-engage-paella-player-7
```

The offending command is:

https://github.com/opencast/opencast/blob/2247c0117426d4e8e36e31bc331ebd4f92f85430/modules/engage-paella-player-7/pom.xml#L46-L47

But the other commands are likely also incorrect. It seems like this was just copied from somewhere else and never actually tested.

--- a/modules/engage-paella-player-7/package.json
+++ b/modules/engage-paella-player-7/package.json
@@ -4,7 +4,7 @@
   "sourceType": "module",
   "private": true,
   "scripts": {
-    "build": "webpack --mode production --progress",
+    "build": "webpack --mode production",
     "dev": "webpack serve --mode development --host=0.0.0.0",
     "eslint": "eslint --config .eslintrc.js src --resolve-plugins-relative-to .",
     "html-linter": "html-linter --config ../../docs/checkstyle/html-linter.json 'src/**/*.html'",

--- a/modules/engage-paella-player-7/pom.xml
+++ b/modules/engage-paella-player-7/pom.xml
@@ -32,7 +32,9 @@
                 </goals>
                 <configuration>
                   <executable>npm</executable>
-                  <arguments>ci</arguments>
+                  <arguments>
+                    <argument>ci</argument>
+                  </arguments>
                 </configuration>
               </execution>
 
@@ -44,20 +46,24 @@
                 </goals>
                 <configuration>
                   <executable>npm</executable>
-                  <arguments>run check</arguments>
+                  <arguments>
+                    <argument>run</argument>
+                    <argument>check</argument>
+                  </arguments>
                 </configuration>
               </execution>
 
               <execution>
+                <phase>validate</phase>
                 <id>frontend build</id>
                 <goals>
                   <goal>exec</goal>
                 </goals>
-                <phase>compile</phase>
                 <configuration>
-                  <executable>node_modules/gulp/bin/gulp.js</executable>
+                  <executable>npm</executable>
                   <arguments>
-                    <argument>paella-opencast:build</argument>
+                    <argument>run</argument>
+                    <argument>build</argument>
                   </arguments>
                 </configuration>
               </execution>
@@ -113,8 +119,8 @@
               </execution>
 
               <execution>
-                <phase>process-resources</phase>
-                <id>npm run build</id>
+                <phase>validate</phase>
+                <id>frontend build</id>
                 <goals>
                   <goal>npm</goal>
                 </goals>

